tests/tally/generator.rs

```diff
@@ -313,24 +326,20 @@ impl VoteRoundGenerator {
             committee_end,
         );
 
-        if let VoteTallyPayload::Private { .. } = payload {
-            let encrypted_tally_fragment =
-                fragment_builder.encrypted_tally(committee_member, voteplan);
-            committee_member.confirm_transaction();
-            res.push(encrypted_tally_fragment);
-        }
-
         let tally_fragment = fragment_builder.vote_tally(committee_member, voteplan, payload);
         committee_member.confirm_transaction();
         res.push(tally_fragment);
 
         res
     }
 
-    pub fn statuses(&mut self) -> Vec<VotePlanStatus> {
+    pub fn statuses(&mut self, ledger: &'a Ledger) -> Vec<VotePlanStatus> {
+        let token_totals = self.token_totals(ledger);
         self.voteplan_managers
             .values()
-            .map(|manager| manager.statuses())
+            .map(|manager| {
+                manager.statuses(TokenDistribution::new(&token_totals, ledger.accounts()))
```

Comment:
```suggestion
                manager.statuses(ledger.token_distribution())
```

Reply:
although, to be fair, I think this will need to removed in https://github.com/input-output-hk/chain-libs/pull/756 so it doesn't really matter that much (I mean, the argument will  not be needed anymore)